charts/gardener-extension-provider-aws/templates/configmap.yaml

```diff
@@ -24,3 +24,7 @@ data:
 {{- if .Values.config.etcd.backup }}
 {{ toYaml .Values.config.etcd.backup | indent 6 }}
 {{- end }}
+{{- if .Values.config.shootStorageClassConfig.encrypted }}
+    shootStorageClassConfig:
+      encrypted: true
```

Comment:
```suggestion
{{- if .Values.config.shootStorageClassConfig }}
    shootStorageClassConfig:
      encrypted: {{ .Values.config.shootStorageClassConfig.encrypted }}
```

Reply:
The default value is `false` so we should not update the resource if not needed.